server/src/features/CodeLensRequestHandler.ts

```diff
@@ -22,7 +22,10 @@ export function codeLensRequestHandler(definitionsProvider: YmlDefinitionProvide
             .map((func: YmlFunction) => {
                 const lens = CodeLens.create(func.definitionLocation.range);
                 lens.command = {
-                    title: `Cognitive complexity is ${func.getCognitiveComplexity()}.`,
+                    // Add a heart symbol when the complexity is equal or below 20.
+                    title: `${
+                        func.getCognitiveComplexity() <= 20 ? '$(heart) ' : ''
```

Comment:
20 seems kinda arbitrary and large xD

Reply:
In fact, according to [this post](https://community.sonarsource.com/t/how-to-use-cognitive-complexity/1894/4) it completely depends on the language.
For YML, I judged it high enough but not too high.